### ISSUE TITLE
Unable to run Node 

### ISSUE THREAD

**Issue by randomite** [babun/babun#216]
I wanted to see if Node would work, but when I tried it gave me this error:
````Shell
{ ~ }  » node

net.js:156
    this._handle.open(options.fd);
                 ^
Error: EINVAL, invalid argument
    at new Socket (net.js:156:18)
    at process.stdin (node.js:664:19)
    at startup (node.js:145:16)
    at node.js:906:3
````

But Node works in Powershell, Cygwin, and Command Prompt: 

![babun 2](https://cloud.githubusercontent.com/assets/6154064/5443981/5b4eabda-845b-11e4-8d11-bba56ad4d52b.PNG)

**Comment by prochafilho** [comment]
Could you distill the instructions on how to run console2 with babun? Im not sure how to make it work to run nodejs...

**Comment by jlupi** [comment]
@randomite we just released babun 1.1.0. It sould work in this version. Just execute  'babun update' and wait until it's done. Then:
````
pact install gcc-g++
wget http://nodejs.org/dist/node-v0.4.12.tar.gz
tar xvfz node-v0.4.12.tar.gz
cd node-v0.4.12
./configure
make
make install
node
````

![image](https://cloud.githubusercontent.com/assets/4189968/6991981/cd8e3ff4-dab3-11e4-8a6f-c79a8e698ef1.png)

**Comment by randomite** [comment]
I am not having too much luck, it seems like updating broke a few things for me:

![image](https://cloud.githubusercontent.com/assets/6154064/6994190/e237e796-dac4-11e4-8943-4d3ce7f16978.png)

**Comment by jlupi** [comment]
yikes! Thats ugly. Looks like babun/cygwin update failed for you. Did you notice any errors during the update. Did babun restart itself after the update? Would be great to habe a screen of the error. 

Nevertheless im affraid you need a fresh install.

**Comment by jlupi** [comment]
Looks like the node problem itself was fixed in 1.1.0. Feel free to reopen if you still have the problem.

**Comment by reinert** [comment]
build from source does not work with [latest version](http://nodejs.org/dist/node-latest.tar.gz)
